test/org/loklak/harvester/RedirectUnshortenerTest.java

```diff
@@ -21,7 +21,7 @@ public void testRedirectUnshortener() {
         shortlinkMap.put("http://ow.ly/P9SX30ddSTI", "https://github.com/loklak/loklak_server/issues/1284");
         shortlinkMap.put("http://bit.do/blog-fossasia", "http://blog.fossasia.org/");
         shortlinkMap.put("http://fb.me/4lcXZsyyO", "https://www.facebook.com/permalink.php?story_fbid=1550262581900846&id=1381813572079082");
-        shortlinkMap.put("http://wp.me/sf2B5-shorten", "https://en.blog.wordpress.com/2009/08/14/shorten/");
+        shortlinkMap.put("http://wp.me/sf2B5-shorten", "http://en.blog.wordpress.com/2009/08/14/shorten/");
```

Comment:
Earlier (before this patch), the redirections were resolved until a final URL is reached. But it isn't required as the prime reason behind resolving redirects is to avoid tracking by other web services.

Once the URL is no longer a threat to anonymity, there is no need to resolve redirects any further as it will take time for no reason.

In this case, `http://en.blog...` is not a trackable URL so we'll not get redirected to `https://en.blog...`, so I changed it here.